### PR TITLE
Add support for rollbar_username

### PR DESCRIPTION
**Pull request by jackton1**
@basoko

### CODE REVIEW

**Comment by basoko** [comment]
Hi @jackton1 !

Thanks for contributing to this! I think that as `status` is optional, if we use an input parameter for the `rollbar_username` we can't know if the `$3` value if for `status` or `rollbar_username`. 

I think that we could add it as environment variable. Could you please make that change to be used from an env var?

Thanks!

**Comment by jackton1** [comment]
Sure

- [ ] Update docs.

**Comment by jackton1** [comment]
> Hi @jackton1 !
> 
> Thanks for contributing to this! I think that as `status` is optional, if we use an input parameter for the `rollbar_username` we can't know if the `$3` value if for `status` or `rollbar_username`.
> 
> I think that we could add it as environment variable. Could you please make that change to be used from an env var?
> 
> Thanks!

Done @basoko

File: README.md [hunk 1]
```diff
@@ -42,6 +45,7 @@ steps:
       version: ${{ github.sha }}
     env:
       ROLLBAR_ACCESS_TOKEN: ${{ secrets.ROLLBAR_ACCESS_TOKEN }}
+      ROLLBAR_USERNAME: ${{ secrets.ROLLBAR_USERNAME }}
```
Comment (basoko) [inline README.md:48]: One question for the shake of curiosity, for your use case do you plan to always use the same username when sending a deploy notification? 
I don't know if you tried, but maybe you could use the `github.actor`, the documentation says:

```
github.actor | string | The login of the user that initiated the workflow run.
``` 

So, in that way, the username send to Rollbar would be dynamic, what do you think?

Comment (jackton1) [reply README.md:48]: Yeah was thinking the same thing. I guess we can update it would also recommend that the username matches rollbar user.

**Pull request merged**